packages/app-desktop/gui/NoteList/NoteList2.tsx

```diff
@@ -198,7 +198,7 @@ const NoteList = (props: Props) => {
 
 	const renderEmptyList = () => {
 		if (props.notes.length) return null;
-		return <div className="emptylist">{getEmptyFolderMessage(props.folders, props.selectedFolderId)}</div>;
+		return <div className="emptylist" role="status">{getEmptyFolderMessage(props.folders, props.selectedFolderId)}</div>;
```

Comment:
(Optional) Consider leaving a comment that explains why `role='status'` is used here.